source/util/StringUtils.h

```diff
@@ -39,6 +40,23 @@ namespace Aws
                  * @return the sanitized value
                  */
                 std::string Sanitize(std::string value);
+
+                /**
+                 * \brief helper method to concat strings with ':' in between
+                 *
+                 * @param first first string
+                 * @param second second string
+                 * @return string in 'first:second' form
+                 */
+
+                std::string addString(Aws::Crt::String first, Aws::Crt::String second);
+                /**
+                 * \brief Given an input map object, will return the string form of the map
```

Comment:
is it in JSON format? Might be helpful to state whether this is a valid JSON object or not

Reply:
The input coming from FP template is validated to be JSON by the API. So before the response is available for us, Device Config is validated to be in JSON format and then converted into Map object.

Reply:
Ok, but is your function generating a valid JSON object? That might be important information for someone reading the API documentation here

Reply:
No, it will return String form of the map object. I thing the method name can confuse users, will update that to 'MapToString'

A good example can is this [maptoString](https://github.com/awslabs/aws-iot-device-client/pull/141/files#diff-e68e49b78fd339a42c1c8b65419a470b5d3af97731474c4ba5c3a02ac0f494fe) Unit test I have added in TestStringUtils.cpp file.